.vib/supabase-postgres-meta/goss/supabase-postgres-meta.yaml

```
@@ -0,0 +1,6 @@
+command:
+  list-run-operations:
+    exec: cd /opt/bitnami/supabase-postgres-meta; npm run
+    exit-status: 0
+    stdout:
+      - node dist/server/server.js
```

I would maybe directly run the server instead of sowing the available commands, wdyt?

```suggestion
    exec: cd /opt/bitnami/supabase-postgres-meta; timeout --preserve-status 5 npm run start || true
    exit-status: 0
    stdout:
      - node dist/server/server.js
      - Server listening
```

---

A much better approach to test this, thanks!